### PR TITLE
Add install guide

### PR DESCRIPTION
**Pull request by kra-mo**

### CODE REVIEW

File: docs/installation/index.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+## Creating a bootable USB stick
+This section will guide you through creating a bootable USB stick with Vanilla OS on it.
+
+### Downloading Vanilla OS
+First, download the Vanilla OS disk image that will be written to your USB flash drive from [here](https://vanillaos.org/).
+
+### Installing Etcher
+To flash the disk image to your USB stick, download and install [balenaEtcher](https://www.balena.io/etcher/).
+
+If you wish, you can also use different software that you are familiar with, such as [Ventoy](https://www.ventoy.net/) or  [Rufus](https://rufus.ie/).
```
Comment (kbdharun) [inline docs/installation/index.md:22]:
```suggestion
If you wish, you can also use different software that you are familiar with, such as [Ventoy](https://www.ventoy.net/) or [Rufus](https://rufus.ie/).
```

**Review by kbdharun — COMMENTED** [review]
Just a small suggestion. I will review this later today. Let me know if you need any help with proofreading.

File: docs/installation/index.md (hunk 1 above)
Comment (kra-mo) [reply docs/installation/index.md]: Thanks, I would appreciate that. I'm currently in the process of adding screenshots.

File: docs/installation/index.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+Open GParted and see if you have a partition that reads "EFI System Partition" \(if you have a modern computer, you most likely will\). If so, select `/boot/efi` as its mount point and leave the format as "Do not touch".
+
+Click "Apply" and review your changes.
+
+Create your user account and review the changes that will be made to your system. Click on "Install Vanilla OS" and the installation will begin.
+
+![Confirming the installation](/assets/uploads/installer-confirm-installation.webp)
+
+## Advanced installation
+This section will help you with manual partitioning in case your particular needs for installation aren't covered by this guide.
```
[the remark this answers is not in the document]
Comment (kra-mo) [reply docs/installation/index.md:114]: How about "this guide doesn't cover your particular needs for installation"?

File: docs/installation/index.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+Select the disk that you want to install Vanilla OS to and click "Configure". Select "Manual Partitioning" and click on the arrow next to it. Click on "Open GParted" to manually partition your disk.
```
Comment (kbdharun) [inline docs/installation/index.md:116]:
```suggestion
Select the disk to which you want to install Vanilla OS and click "Configure". Select "Manual Partitioning" and click on the arrow next to it. Click on "Open GParted" to manually partition your disk.
```

File: docs/installation/index.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+**Warning**: this will erase **all data** on your USB flash drive. Make sure to back up all your important data before proceeding!
+
+Open Etcher, select the downloaded disk image and your USB flash drive, then click "Flash!".
+
+![Flashing](/assets/uploads/installation-flashing.webp)
+
+When the process is done, you will have a bootable USB stick with Vanilla OS on it.
+
+## Booting from the USB flash drive
+Insert the USB flash drive in the computer that you want to install Vanilla OS to and power up or restart the device.
```
Comment (kbdharun) [inline docs/installation/index.md:34]:
```suggestion
Insert the USB flash drive into the computer to which you want to install Vanilla OS and power up or restart the device.
```

File: docs/installation/index.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+If your device does not automatically boot into the Vanilla OS installation media, reboot, and while the device is starting up, hold down the key to `select boot device` or for `boot menu` indicated by a prompt on the boot screen. The most common keys are F2, F10, F11, F12, Delete and Escape. If the prompt isn't present, try searching online for your specific model. Select Vanilla OS or the name of your USB flash drive from the boot menu.
+
+When your device has succesfully booted from the USB stick, you should see a welcome screen inviting you to either try or install Vanilla OS.
```
Comment (kbdharun) [inline docs/installation/index.md:38]:
```suggestion
When your device has successfully booted from the USB stick, you should see a welcome screen inviting you to try or install Vanilla OS.
```

File: docs/installation/index.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+Open Etcher, select the downloaded disk image and your USB flash drive, then click "Flash!".
+
+![Flashing](/assets/uploads/installation-flashing.webp)
+
+When the process is done, you will have a bootable USB stick with Vanilla OS on it.
+
+## Booting from the USB flash drive
+Insert the USB flash drive in the computer that you want to install Vanilla OS to and power up or restart the device.
+
+If your device does not automatically boot into the Vanilla OS installation media, reboot, and while the device is starting up, hold down the key to `select boot device` or for `boot menu` indicated by a prompt on the boot screen. The most common keys are F2, F10, F11, F12, Delete and Escape. If the prompt isn't present, try searching online for your specific model. Select Vanilla OS or the name of your USB flash drive from the boot menu.
```
Comment (kbdharun) [inline docs/installation/index.md:36]:
```suggestion
Suppose your device does not automatically boot into the Vanilla OS installation media reboot it. While it is starting up, hold down the key to `select boot device` or `boot menu` indicated by a prompt on the boot screen. The most common keys are F2, F10, F11, F12, Delete and Escape. If the prompt isn't present, try searching online for your specific model. Select Vanilla OS or the name of your USB flash drive from the boot menu.
```

File: docs/installation/index.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+
+If you wish to install Vanilla OS on the same computer, but on a different drive than Windows, follow the steps described in [Installing only Vanilla OS](/docs/installation/#title8).
+
+### Opening GParted
+Select the disk that Windows is installed on and click "Configure". Select "Manual Partitioning" and click on the arrow next to it. Click on "Open GParted" to manually partition your disk.
+
+![Manual Partitioning](/assets/uploads/installer-manual-partitioning.webp)
+
+### Resizing your C: drive
+Once you are in GParted, you will see a visual representation of your disks' partitions. In the top right corner, select the disk that Windows is installed on. Select the `ntfs` partition equivalent to your Windows `C:` drive (usually the biggest one). If you are unsure which one that is, reboot to Windows, open Disk Management and check the partitions against the ones displayed here.
```
Comment (kbdharun) [inline docs/installation/index.md:79]:
```suggestion
Once you are in GParted, you will see a visual representation of your disks' partitions. In the top right corner, select the disk that Windows is installed on. Select the `NTFS` partition equivalent to your Windows `C:` drive (usually the biggest one). If you are unsure which one that is, reboot to Windows, open Disk Management and check the partitions against the ones displayed here.
```

File: docs/installation/index.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+
+Right click on the selected partition and click "Resize/Move". Here, you can drag either edge of the partition to make it smaller. You can also configure the exact new size of the partition at the bottom.
```
Comment (kbdharun) [inline docs/installation/index.md:81]:
```suggestion
Right-click on the selected partition and click "Resize/Move". Here, you can drag either edge of the partition to make it smaller. You can also configure the exact new size of the partition at the bottom.
```

File: docs/installation/index.md [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+
+![Shrinking your C: drive](/assets/uploads/installation-shrink-c.webp)
+
+Once you shrank it by how much space you want to take from Windows and dedicate to Vanilla OS (at least 25GB), click "Resize/Move".
+
+### Creating the new partitions
+After shrinking your Windows partition, you will be left with unallocated space. You will have to create 2 partitions in this space. Right click on it and select "New".
```
Comment (kbdharun) [inline docs/installation/index.md:88]:
```suggestion
After shrinking your Windows partition, you will be left with unallocated space. You will have to create 2 partitions in this space. Right-click on it and select "New".
```

**Review by kbdharun — COMMENTED** [review]
I have some minor suggestions for this PR.
> I will go through this again tomorrow.

File: docs/installation/index.md (hunk 7 above)
Comment (kra-mo) [reply docs/installation/index.md:79]: Filesystem types are lowercase in GParted

File: docs/installation/index.md (hunk 6 above)
Comment (kra-mo) [reply docs/installation/index.md:36]: Suppose seems too formal and you missed a comma, but "device" shouldn't be repeated

**Comment by kra-mo** [comment]
Moved to the handbook

**Pull request closed without merging**